tsfile/src/main/java/org/apache/iotdb/tsfile/write/chunk/ChunkWriterImpl.java

```diff
@@ -80,7 +80,7 @@
   private Statistics<?> statistics;
 
   private boolean isSdtEncoding;
-
+  private boolean isLastPoint;
```

Comment:
add comment: 
used for SDT. When the ChunkWriter WILL write the last data point in the chunk, set it to true to tell SDT saves the point.